.github/workflows/build-readiness-image.yml

```diff
@@ -0,0 +1,62 @@
+name: Readiness Image Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+  packages: write
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: readiness
+
+jobs:
+  build-llm-d-readiness:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Log in to Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
```

Comment:
Given the choices here for username and password, the registry is not a free variable.

Reply:
In other words, the registry has to be GitHub's. Having a variable gives only an illusion of choice.